frontend/gotgt/frontend.go

```diff
@@ -38,10 +40,23 @@ type goTgt struct {
 	stats        scsi.Stats
 }
 
+var _ types.Frontend = (*goTgt)(nil)
+var _ api.RemoteBackingStore = (*controller.Controller)(nil)
+
+func initializeSCSITarget(size int64) {
+	iscsit.EnableStats = true
+	scsi.SCSIVendorID = "CLOUDBYTE"
+	scsi.SCSIProductID = "OPENEBS"
+	scsi.SCSIID = "iqn.2016-09.com.jiva.openebs:iscsi-tgt"
```

Comment:
Is this hardcoded? Will it not have the pv uuid included here. like: `iqn.2016-09.com.openebs.jiva:<uuid>`

Reply:
Also what happens if the above changes are done and jiva volumes from 1.14 are upgraded to 1.15. Is there a way to know if this is a new volume or existing volume -- and apply the changes related to spec changes only for new volumes.

Reply:
t.tgtName is where we include pv uuid, scsiid is something different, which was set in gostor/gotgt
@payes may tell us difference

Reply:
this variable is not being used anywhere